translations/fr/messages.php

```diff
@@ -39,13 +39,18 @@
     'The selected sensor cannot be characterized. Please select another sensor among cameras (all camera types : RGB, multispectral, etc.), spectrometers and LiDAR.' => 'Le capteur sélectionné ne peut pas être caractérisé. Veuillez sélectionner une caméra (RGB, TIR, multispectrale, etc.), un spectromètre ou un LiDAR. ',
     'The alias of the plot (e.g. MAU17-PG_38_WW_1_19_5)' => 'L\'alias du plot (ex. MAU17-PG_38_WW_1_19_5)',
     'The experiment modalities of the plot (e.g. WW, WD)' => 'Les modalités expérimentales du plot (ex. WW, WD)',
+    'The object update has failed. See insertion status column for more details.' => 'La mise à jour des objets a échoué. Voir la colonne status pour plus d\'informations',
     'The replication of the plot (e.g. 2, A)' => 'La répétition du plot (ex. 2, A)',
     'The URI of the scientific object (e.g http://www.phenome-fppn.fr/phenovia/2017/o1028649)' => 'L\'URI de l\'objet scientifique (ex. http://www.phenome-fppn.fr/phenovia/2017/o1028649)',
     'The URI of the experiment (e.g. http://www.phenome-fppn.fr/pheno3c/P3C2017-6)' => 'L\'URI de l\'experimentation (ex. http://www.phenome-fppn.fr/pheno3c/P3C2017-6)',
     'The URI of the species (e.g. http://www.phenome-fppn.fr/id/species/zeamays)' => 'L\'URI de l\'espèce (ex. http://www.phenome-fppn.fr/id/species/zeamays)',
     'The error above occurred while the Web server was processing your request.' => 'L\'erreur ci-dessus est survenue lors du traitement de votre requête par le serveur web.',
     'The value' => 'La valeur',
     'The variety used in the plot (e.g. apache)' => 'La variété du plot (ex. apache)',
+    'To change the value of one property (e.g. modality), you need to provide all of its former properties that you want to keep, otherwise this information will be lost.' => 'Pour changer la valeur d\'une propriété (ex : modalité), vous devez renseigner toutes les propriété de l\'objet scientifique. Sinon, les propriétés vides seront considérées comme des valeurs à supprimer et de l\'information sera perdue.',
```

Comment:
'Be carefull to write all the new and the former properties of the updated object.' => 'Attention: veillez à écrire toutes les propriétés nouvelles et anciennes de l'objet mis à jour.',